README.md

```diff
@@ -34,37 +32,51 @@ Sends a deployment complete message to a Slack channel when [Trellis](https://gi
 - Valid Slack Webhook URL
 
 ## Screenshots
+
 ![Deployment Screenshot](deployment-screenshot.png)
 
 ## Installation
 
 Add this role to `galaxy.yml`:
+
 ```yaml
 # galaxy.yml
-- src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
-  version: 0.1.0 # Check for latest version!
+  src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
+  version: 0.5.0 # Check for latest version!
```

Comment:
```suggestion
- src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
  version: 0.1.0 # Check for latest version!
```
The version number has not yet changed and we need to keep the hyphen for valid yml syntax

Reply:
No probs, we forked off so long + been relying on this repo / versioning - once it's merged back into your repo should be cool after you bump the release